events/events-team/content/new-contributor-workshops/labels-and-bots.md

```diff
@@ -11,7 +11,7 @@ This task is meant as a short overview for Intermediate content.
 
 ### Slides
 
-The [PR workflow diagram](git.k8s.io/community/contributors/guide/git_workflow.png) is useful here. There are also previous slides that explain the differnet labels.
+The [PR workflow diagram](https://github.com/kubernetes/community/blob/master/contributors/guide/git_workflow.png) is useful here. There are also previous slides that explain the differnet labels.
```

Comment:
For git repo links we tend to prefer the "vanity url" redirector.  The links are shorter to read.